src/state/GlobalState.tsx

```diff
@@ -460,6 +460,20 @@ export class GlobalState extends PureComponent<Props> {
       (game) => game.appName === appName
     )[0]
 
+    // Update recently played games if the game was just launched
```

Comment:
I think you can remove this comment, this doesn't look hacky to me.
The `configstore.onDidChange` was giving me false positives in the past. I was using it to update the game status on the frontend some time ago.

Reply:
> this doesn't look hacky to me

Changing something (seemingly) unrelated to this (the "setGameStatus" IPC call) will break this functionality. This shouldn't be the case, so it's a hack IMO.